com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py

```diff
@@ -720,6 +721,37 @@ def end_parallel(self):
         endP = Stream(self.topology, oport)
         return endP
 
+    def last(self, size=1):
```

Comment:
On second thought, this doesn't keep a separation between the contents of the window and the details of its aggregation.